notification/slack.py

```diff
@@ -166,13 +167,28 @@
 OLD_SLACK_INCOMING_WEBHOOK = 'https://%s/services/hooks/incoming-webhook?token=%s'
 SLACK_INCOMING_WEBHOOK = 'https://hooks.slack.com/services/%s'
 
+import urllib
+
+# See https://api.slack.com/docs/message-formatting#how_to_escape_characters
+html_escape_table = {
+    '&': "&amp;",
+    '>': "&gt;",
+    '<': "&lt;",
+    '"': "\"",
```

Comment:
I think you should add a comment here, saying that `"` and `'` are not discussed by the link to the slack api above, but are necessary because of the way Ansible (?) quoting works.